mappings/net/minecraft/block/Blocks.mapping

```diff
@@ -29,9 +29,17 @@ CLASS net/minecraft/unmapped/C_qceyzrkd net/minecraft/block/Blocks
 		ARG 0 litLevel
 	METHOD m_idzyfosb (Lnet/minecraft/unmapped/C_txtbiemp;)I
 		ARG 0 state
+	METHOD m_ihepvmyd createPillarColor (Lnet/minecraft/unmapped/C_mlrprqox;Lnet/minecraft/unmapped/C_mlrprqox;Lnet/minecraft/unmapped/C_aevintex;)Lnet/minecraft/unmapped/C_djwplmtn;
```

Comment:
maybe `createPillarMapColor`? Just color is a bit ambiguous.

Reply:
*facepalms*
i might have wanted a `createPillarBlock` here, whoops